src/unix/process.c

```diff
@@ -224,6 +224,52 @@ static int uv__process_init_stdio(uv_stdio_container_t* container, int fds[2]) {
   }
 }
 
+#ifdef __MVS__
+/* TODO: Revisit once V2R5 Framework + CLIB Override lands in ZOSLIB. */
+static int os390_process_init_stdio(uv_stdio_container_t* container,
```

Comment:
I don't want too much platform-specific code in this file so please move it to src/unix/os390.c and prepend `uv__` to the function name.